src/libs/runtime-utils.js

```diff
@@ -261,11 +261,20 @@ export const getCurrentAttachedDataDisk = (app, appDataDisks) => {
 }
 
 // If the disk was attached to an app, return the appType. Otherwise return undefined.
-export const getDiskAppType = disk => {
-  const saturnApp = disk.labels.saturnApplication
+export const getDiskAppType = ({ labels: { saturnApplication } }) => {
   // Do a case-insensitive match as disks have been created with both "galaxy" and "GALAXY".
-  const appType = _.find(type => type.toLowerCase() === saturnApp?.toLowerCase(), allAppTypes)
-  return appType
+  return _.find(type => type.toLowerCase() === saturnApplication?.toLowerCase(), allAppTypes)
+}
+
+export const haveTooManyDisks = (appType, filteredDisks, disksByProject) => {
```

Comment:
Since this is being exposed as a method, and it's complicated, please add unit test coverage.